### PR TITLE
Basic GoLang highlighting.

### PR DESCRIPTION
**Pull request by elliotcourant**
I've started adding some very basic GoLang highlighting. Please provide some feedback. 

alt text

### CODE REVIEW

File: colors/material.vim [hunk 1]
```diff
@@ -189,6 +189,21 @@ call s:HL('jsTemplateExpression', s:red, '', '')
 call s:HL('jsThis', s:red, '', '')
 call s:HL('jsUndefined', s:orange, '', '')
 
+" Golang 
+call s:HL('goComment', s:gray5, '', '')
+call s:HL('goFunctionCall', s:blue, '', 'bold')
```
Comment (kaicataldo) [inline colors/material.vim]: I personally have a preference not to use bold text for syntax highlighting (it's used to highlight bold text in markdown files and for some Vim-specific features).

**Review by kaicataldo — COMMENTED** [review]
Hey, thanks for the PR! Do you mind sharing a screenshot of what it looks like with what's currently in master and with your changes?

File: colors/material.vim [hunk 2]
```diff
@@ -189,6 +189,21 @@ call s:HL('jsTemplateExpression', s:red, '', '')
 call s:HL('jsThis', s:red, '', '')
 call s:HL('jsUndefined', s:orange, '', '')
 
+" Golang 
+call s:HL('goComment', s:gray5, '', '')
```
Comment (kaicataldo) [inline colors/material.vim]: Is there a reason [this line](https://github.com/kaicataldo/material.vim/pull/11/files#diff-693a3fb1845dd2c150138d65ec2fe8d6R101) doesn't cover comments in Golang? What Golang language plugin are you using?

Also, would you mind updating it to `s:gray4` so it matches?

Comment (elliotcourant) [reply colors/material.vim]: I'm currently using `vim-go` but that highlighting is probably redundant. I'll remove it and see if it still works properly, if not I'll update it to `s:gray4`

File: colors/material.vim (hunk 1 above)
Comment (elliotcourant) [reply colors/material.vim]: I'll tweak that!

**Comment by elliotcourant** [comment]
I ended up tweaking the comment color to `gray4`, and I've removed the bold. I will post a screenshot of the new colors that I've added and the current colors without my changes here shortly. Thank for the feedback so far!

**Comment by elliotcourant** [comment]
image

This is how it looks without my changes. My changes were trying to match https://plugins.jetbrains.com/plugin/8006-material-theme-ui as closely as I could.

**Comment by kaicataldo** [comment]
Thanks for contributing!

**Pull request merged**